diagnostic.md

```diff
@@ -34,21 +34,21 @@ Open up this file in Atom, and write your answers below (where indicated).
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+Relative path - the described path is in reference to the current working directory
```

Comment:
Absolute path. When a path starts with `/` it means its from the root directory indicating an Absolute path.